docs/source/docs_guide.md

```diff
@@ -0,0 +1,212 @@
+
+## Testing your changes
+
+You are strongly encouraged to test your changes to the documentation before you
+submit a PR. You should start by building the docs on your own machine, and
+subsequently push your changes to your own GitHub staging repo where they can
+populate your [ReadTheDocs](https://readthedocs.org/) publication website. Once
+you are happy with your change, you can submit it via a PR for inclusion in the
+main Fabric repository.
+
+The following sections cover first how to build the docs locally, and then
+use your own Github fork to publish on ReadTheDocs.
+
+## Building locally
+
+Once you've cloned the Fabric [repository]() to your local machine, use these
+quick steps to build the Fabric documentation on your local machine. Note: you
+may need to adjust depending on your OS.
+
+Prereqs:
+ - [Python 3.7](https://wiki.python.org/moin/BeginnersGuide/Download)
+ - [Pipenv](https://docs.pipenv.org/en/latest/#install-pipenv-today)
+
+```
+cd fabric/docs
+pipenv install
+pipenv shell
+make html
+```
+
+This will generate all Fabric documentation html files in `docs/build/html`
+which you can then start browsing locally using your browser; simply navigate to
+the `index.html` file.
+
+Make a small change to a file, and rebuild the documentation to verify that your
+change was built locally. Every time you make a change to the documentation you
+will of course need to rerun `make html`.
+
+In addition, if you'd like, you may also run a local web server with the
+following commands (or equivalent depending on your OS):
+
+```
+sudo apt-get install apache2
+cd build/html
+sudo cp -r * /var/www/html/
+```
+
+You can then access the html files at `http://localhost/index.html`.
+
+## Building on GitHub
+
+It can often be helpful to use your fork of the Fabric repository to build the
+Fabric documentation in a public site, available to others. The following
+instructions show you how to use ReadTheDocs to do this.
+
+1. Go to http://readthedocs.org and sign up for an account.
+2. Create a project. Your username will preface the URL and you may want to
+   append `-fabric` to ensure that you can distinguish between this and other
+   docs that you need to create for other projects. So for example:
+   `YOURGITHUBID-fabric.readthedocs.io/en/latest`.
+3. Click `Admin`, click `Integrations`, click `Add integration`, choose `GitHub
+   incoming webhook`, then click `Add integration`.
+4. Fork [Fabric on GitHub](https://github.com/hyperledger/fabric).
+5. From your fork, go to `Settings` in the upper right portion of the screen.
+6. Click `Webhooks`.
+7. Click `Add webhook`.
+8. Add the ReadTheDocs's URL into `Payload URL`.
+9. Choose `Let me select individual events`:`Pushes`、`Branch or tag creation`、
+   `Branch or tag deletion`.
+10. Click `Add webhook`.
+
+Now anytime you modify or add documentation content to your fork, this
+URL will automatically get updated with your changes!
+
+## Commands Reference updates
```

Comment:
Documenting commands looks like something that may need to be handled differently in other languages

Reply:
I'll add a caveat that says that this is the process for English language docs. Other languages to come. (I do hope we've not done anything that hinders that, but right now it's not a major issue.)